microprofile.jdt/com.redhat.microprofile.jdt.test/src/main/java/com/redhat/microprofile/jdt/core/ProjectLabelTest.java

```diff
@@ -82,31 +82,25 @@ public void getProjectLabelMultipleProjects() throws Exception {
 	public void projectNameMaven() throws Exception {
 		IJavaProject quarkusMaven = BasePropertiesManagerTest.loadMavenProject(MavenProjectName.using_vertx);
 		IJavaProject maven = BasePropertiesManagerTest.loadMavenProject(MavenProjectName.empty_maven_project);
-		IJavaProject folderNameDifferent = BasePropertiesManagerTest.loadMavenProject(MavenProjectName.folder_name_different_maven);
 		List<ProjectLabelInfoEntry> projectLabelEntries = ProjectLabelManager.getInstance().getProjectLabelInfo();
 		assertName(projectLabelEntries, quarkusMaven, "using-vertx");
 		assertName(projectLabelEntries, maven, "empty-maven-project");
-		assertName(projectLabelEntries, folderNameDifferent, "mostly.empty");
 	}
-
+	
 	@Test
-	public void projectNameSameFolderName() throws Exception {
-		IJavaProject empty1 = BasePropertiesManagerTest.loadMavenProject(MavenProjectName.empty_maven_project);
-		IJavaProject empty2 = BasePropertiesManagerTest.loadMavenProjectFromSubFolder(MavenProjectName.other_empty_maven_project, "folder");
+	public void projectNameMavenDifferentName() throws Exception {
+		IJavaProject folderNameDifferent = BasePropertiesManagerTest.loadMavenProject(MavenProjectName.folder_name_different_maven);
 		List<ProjectLabelInfoEntry> projectLabelEntries = ProjectLabelManager.getInstance().getProjectLabelInfo();
-		assertName(projectLabelEntries, empty1, "empty-maven-project");
-		assertName(projectLabelEntries, empty2, "other-empty-maven-project");
+		assertName(projectLabelEntries, folderNameDifferent, "mostly.empty");
 	}
 
 	@Test
 	public void projectNameGradle() throws Exception {
 		IJavaProject quarkusGradle = BasePropertiesManagerTest.loadGradleProject(GradleProjectName.quarkus_gradle_project);
 		IJavaProject gradle = BasePropertiesManagerTest.loadGradleProject(GradleProjectName.empty_gradle_project);
-		IJavaProject renamedGradle = BasePropertiesManagerTest.loadGradleProject(GradleProjectName.renamed_quarkus_gradle_project);
 		List<ProjectLabelInfoEntry> projectLabelEntries = ProjectLabelManager.getInstance().getProjectLabelInfo();
 		assertName(projectLabelEntries, quarkusGradle, "quarkus-gradle-project");
 		assertName(projectLabelEntries, gradle, "empty-gradle-project");
-		assertName(projectLabelEntries, renamedGradle, "my-gradle-project");
```

Comment:
When loading Gradle projects (ie, from VS Code), if the folder name is `folder-name` and the project name in .project is `new-name`, unlike loading Maven projects, I've observed that the IResource path is `P/new-name` instead of `P/folder-name`.

Since Maven projects are being focussed on, for now, I removed the Gradle test case where folder name != project name.

Reply:
This removed test case will cause a NPE since thid PR loads projects using folder name: `JavaModelManager.getJavaModelManager().getJavaModel().getJavaProject({folder name here});` in BasePropertiesManagerTest.java.